<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>2084</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhotoStall</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu21">
    <property name="title">
     <string>&amp;文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;编辑</string>
    </property>
    <addaction name="action_back"/>
    <addaction name="action_recover"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;视图</string>
    </property>
    <addaction name="action_geometry"/>
    <addaction name="action_special"/>
    <addaction name="action_paint"/>
    <addaction name="action__layer"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>&amp;帮助</string>
    </property>
   </widget>
   <addaction name="menu21"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveas"/>
   <addaction name="action_back"/>
   <addaction name="action_recover"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dock_layer">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>图层</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9"/>
  </widget>
  <widget class="QDockWidget" name="dock_paint">
   <property name="minimumSize">
    <size>
     <width>98</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>900</height>
    </size>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>工具箱</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QPushButton" name="pushButton_choose">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/move.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/line.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ciecle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/ellipse.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_rectangle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/rectangle.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/cut.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mosaic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/mosaic.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_color">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_translation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/保存.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/translation.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="combox_pensize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>粗细</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>520</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>模式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_mode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>绘图</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>橡皮擦</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>反橡皮擦</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>660</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_solid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>690</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>空心</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>实心</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_geometry">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>900</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>几何变换</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像大小</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像宽度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_length">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_width">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>px</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像旋转</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_direction">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>顺时针</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>逆时针</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>图像翻转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_turn_h">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>水平翻转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_turn_v">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直翻转</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像长度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_degree">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>°</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confrim_size">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_rotate">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clockwise_90">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>顺时针90°</string>
     </property>
    </widget>
    <widget class="QPushButton" name="anticlockwise_90">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>逆时针90°</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_center">
   <property name="minimumSize">
    <size>
     <width>650</width>
     <height>800</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>图像处理</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dock_special">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>900</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>特殊效果</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>直方图</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>滤镜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_histogram">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>直方图显示</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>滤镜效果</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>滤镜程度</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_filter">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox__filter">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>怀旧</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>黑白</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>图像滤波</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox__filter_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>均值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>高斯滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中值滤波</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双边滤波</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="confirm_filter_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirm_filter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;打开</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;新建</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;保存</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/saveas.png</normaloff>:/saveas.png</iconset>
   </property>
   <property name="text">
    <string>&amp;另存为</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/close.png</normaloff>:/close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;关闭</string>
   </property>
  </action>
  <action name="action_back">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/revert.png</normaloff>:/revert.png</iconset>
   </property>
   <property name="text">
    <string>&amp;撤销</string>
   </property>
  </action>
  <action name="action_recover">
   <property name="icon">
    <iconset resource="icon/icon.qrc">
     <normaloff>:/recover.png</normaloff>:/recover.png</iconset>
   </property>
   <property name="text">
    <string>&amp;重做</string>
   </property>
  </action>
  <action name="action_geometry">
   <property name="text">
    <string>&amp;几何变换</string>
   </property>
  </action>
  <action name="action_special">
   <property name="text">
    <string>&amp;特殊效果</string>
   </property>
  </action>
  <action name="action_paint">
   <property name="text">
    <string>&amp;绘图工具</string>
   </property>
  </action>
  <action name="action__layer">
   <property name="text">
    <string>图层</string>
   </property>
  </action>
  <zorder>dock_geometry</zorder>
  <zorder>dock_center</zorder>
  <zorder>dock_special</zorder>
  <zorder>dock_paint</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
